harvard-lts/mps-viewer
# Adding Jest. Adding test that always passes.

**Adding Jest. Adding test that always passes.**
* * *

**JIRA Ticket**: [LTSVIEWER-103](https://jira.huit.harvard.edu/browse/LTSVIEWER-103)

* Other Relevant Links: https://jestjs.io/

# What does this Pull Request do?
This adds Jest, ~~both globally and~~ through Node Package Manager. It adds a single test `3 equal 3` that always passes.

# How should this be tested?

A description of what steps someone could take to:
* Rebuild the container off of the `add-jest` branch
* ssh into the container: `docker exec -it  mps-viewer bash`
* Run the test using NPM and verify it passes: `npm test`
* ~~Run the test using Jest command line and verify it passes: `jest`~~

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Yes
- integration tests? No

# Interested parties
@dl-maura
